production_configs/20220301_psf_tuned_zen20/readme.md

```diff
@@ -0,0 +1,11 @@
+# New Prod Config 
+
+## 20220301_psf_tuned_zen20
```

Comment:
Thats not what the folder will be named in the end.
Seeing this I think we could improve the PR template a bit. You are the first one to submit, so you notice the initial trouble :)

Reply:
And thanks for this @chaimain,
Could you please update your PR having a look again to the template, please ?

Reply:
I see another "notation problem".
I think that `psf_tuned_zen20` is ok for the sub-title, the prod_id, as @LukasNickel suggested above, is more explicative if it's `dl1ab_tuned_psf`

Reply:
Ok. Will change it to that